spaces/S000101/properties/P000214.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000101
+property: P000214
+value: false
+refs:
+- mathse: 5106079
+  name: Answer to "Do the Arkhangel'skii $\alpha_i$ properties hold for uncountable products?" 
+---
+
+Follows from the answer to {{mathse:5106079}}.
```

Comment:
```suggestion
  name: Do the Arkhangel'skii $\alpha_i$ properties hold for uncountable products?
---

See {{mathse:5106079}}.
```
Here the whole post is relevant for context, not just the answer part.

Same thing for the other files.